services/markets/coingecko/tickers.go

```diff
@@ -114,7 +114,8 @@ func createCgCoinsMap(coins Coins) map[string][]CoinResult {
 				PotentialCoinID: getCoinId(platform),
 			}
 
-			cgCoinsMap[c.Id] = []CoinResult{cr}
```

Comment:
can you write a test for it?

Reply:
This is a little hard to do when writing unit tests？